docs/learn/learn-account-advanced.md

```diff
@@ -330,6 +330,33 @@ derivation paths.
 
 {{ kusama: Note that you cannot import Kusama Ledger accounts in Ledger Live. To see Kusama account balances, you must import your ledger account into a [**wallet**](./wallets). :kusama }}
 
+### Account Derivation on [Subkey](#creating-accounts-with-subkey) and [Polkadot Vault](../general/polkadot-vault.md)
+
+The Subkey tool and Polkadot Vault App use the following Polkadot Standard Hard Derivation scheme:
+
+- `//network` as the primary account for `network`, named according to `network`'s named chain
+  specification
+  - `//network//0`, `//network//1`, ... as the secondary high-security accounts for `network`
+
+Additionally, users can adopt the following good practice scheme:
```

Comment:
We should be more specific that this is a proposal/suggestion, and not strictly necessary.